uwsgi.ini

```diff
@@ -23,7 +23,7 @@ plugin = python3
 max-requests = 1000 # Reload workers after the specified amount of managed requests (avoid memory leaks).
 vacuum = True       # Try to remove all of the generated files/sockets (UNIX sockets and pidfiles) upon exit.
 module =kolibri.deployment.default.wsgi:application
-buffer-size = 8192  # needed to support long requests blocks (in lessons, for example)
+buffer-size = 32768  # needed to support long requests blocks (in lessons, for example)
```

Comment:
Presumably this is only an issue for large API requests, as file requests are streamed?

Reply:
it's thought in long urls (or posts with a lot of parameters, this kind of errors raised especially when accessing to lessons with many resource ids)